md/api.py

```diff
@@ -22,14 +22,7 @@
 
 # PURPOSE_CHOICES to be added after ODPM-31
 
-GROUP_DEFAULTS = {'ASIAN': 0,
-                  'BLACK': 0,
-                  'NATIVE AMERICAN': 0,
-                  'UNKNOWN': 0,
-                  'WHITE': 0,
-                  'HISPANIC': 0,
-                  }
-
+GROUP_DEFAULTS = {k: 0 for k in GROUPS.values()}
```

Comment:
ETHNICITY_CHOICES is a tuple, which breaks this line

`                ethnicity = GROUPS.get(stop['ethnicity'],
                                       stop['ethnicity'])`